contents/javascript/VueJs/Intro.md

```diff
@@ -0,0 +1,161 @@
+  }
+});
+```
+```html
+<!-- VueJs (html) -->
+<div id="array">
+  <span v-for="date in array">{{date}}</span>
+</div>
+```
+
+2. <b>Progressive:</b>
+VueJs is designed from the ground up to be incrementally adoptable. The core library is focused on the view layer only, and is easy to pick up and integrate with other libraries or existing projects. This means that if you have a large application, you can plug VueJs into just a part of your application without disturbing the other components. A quote from Evan You - the founder of VueJs is as follow:
+> Vue.js is a more flexible, less opinionated solution (than Angular). That allows you to structure your app the way you want it to be, instead of being forced to do everything the Angular way (Angular requires a certain way to structure an application, making it hard to introduce Angular into an already built project). It’s only an interface layer so you can use it as a light feature in pages instead of a full blown SPA (single-page application).
+>
+> -- <cite>Evan You</cite> [[source]](https://www.valuecoders.com/blog/technology-and-apps/vue-js-comparison-angular-react/)
+
+3. <b>Versatile:</b>
+VueJs is perfectly capable of powering sophisticated single-page applications when used in combination with modern tooling and supporting libraries.
+
+4. <b>Clean:</b>
+VueJs syntax is simple and this can make the HTML pages very clean. This would allow user interfaces built by VueJs to be more maintainable and testable.
+
+## Drawbacks of VueJS
+
+1. <b>Relatively small size community:</b>
+VueJs is a relatively new JavaScript framework as compared to Angular and React. The size of the community for VueJs is therefore relatively small. Although small size community means you can differentiate yourself from other JavaScript developers, it also means there are fewer resources such as tutorials and problem-shooting guides.
+
+2. <b>Language barriers:</b>
+A majority of users of VueJs are the Chinese as VueJs is developed by a Chinese American. He is supportive of the Chinese community and hence a lot of the existing plugins are written in Chinese. There might be some language barriers for an English speaking developer seeking for VueJs resources.
+
+## Other Advantages and Drawbacks of VueJs
+Here are some of many articles that describe advantages and drawbacks of VueJs:
+- [What is VueJs and What are its Advantages](https://hackernoon.com/what-is-vue-js-and-what-are-its-advantages-4071b7c7993d)
+- [Pros and Cons of the Vue.js framework - Vue.js Feed](https://vuejsfeed.com/blog/pros-and-cons-of-the-vue-js-framework)
+
+Detailed Comparison of VueJs with other JavaScript frameworks can be found from:
+- [Vue Guild: Comparison with Other Frameworks](https://vuejs.org/v2/guide/comparison.html)
+- [Angular vs React vs Vue](https://medium.com/unicorn-supplies/angular-vs-react-vs-vue-a-2017-comparison-c5c52d620176)
+
+## Installation
```

Comment:
This too is not worthy of a separate section. Insert at the start of section `## Creating a Simple Project in VueJs`?